src/sqlfluff/rules/L026.py

```diff
@@ -100,4 +66,84 @@ def _eval(self, context: RuleContext) -> EvalResultType:
         ):
             return LintResult()
 
-        return super()._eval(context=context)
+        violations: List[LintResult] = []
+        start_types = ["select_statement", "delete_statement", "update_statement"]
```

Comment:
I had a look if I could trigger this false positive with the other DML statements too , INSERT and MERGE, but wasn't able to come up with a relevant case off the top of my head so seems fine to not include them here.

Have you also considered these and are happy to not include them in `start_types`?

(worst case they should be easy to add if a case arises)

Reply:
I agree, I think we are okay omitting them.